examples/post.v3.LR.historical.zppy_v3.cfg

```diff
@@ -0,0 +1,293 @@
+  climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  # 3. These sets depend on the ts task's atm_monthly_180x360_aave subtask:
+  # "enso_diags" (not requested), "qbo", "area_mean_time_series" (not requested)
+  obs_ts = '/lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/time-series'
+  # 4. These sets depend on the ts task's atm_daily_180x360_aave subtask:
+  # "tropical_subseasonal"
+  ts_daily_subsection = "atm_daily_180x360_aave"
+  # 5. These sets depend on the ts task's rof_monthly subtask:
+  # "streamflow" (not requested)
+  # 6. These sets depend on the tc_analysis task:
+  # "tc_analysis"
+  # No extra parameters required for this set.
+  #
+  # Plus several other parameters:
+  grid = '180x360_aave'
+  reference_data_path = '/lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/climatology'
+  short_name = 'v3.LR.historical_0051'
+
+  [[atm_monthly_180x360_aave_mvm]]
+  sets="lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","annual_cycle_zonal_mean","qbo","diurnal_cycle","zonal_mean_2d_stratosphere",
+  # Using the groups from above:
+  # Group 1: climo atm monthly
+  # "lat_lon", "zonal_mean_xy", "zonal_mean_2d", "polar", "cosp_histogram", "meridional_mean_2d", "annual_cycle_zonal_mean", "zonal_mean_2d_stratosphere" "aerosol_aeronet" (not requested), "aerosol_budget" (not requested)
+  climo_subsection = "atm_monthly_180x360_aave"
+  reference_data_path = '/lcrc/group/e3sm2/ac.zhang40/E3SMv3/v3.LR.piControl_451-500/post/atm/180x360_aave/clim'
+  # Group 2: climo atm diurnal
+  # "diurnal_cycle"
+  climo_diurnal_frequency = "diurnal_8xdaily"
+  climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  # Group 3: ts atm monthly
+  # "enso_diags" (not requested), "qbo", "area_mean_time_series" (not requested)
+  ts_num_years = 5
+  ts_num_years_ref = 50
+  ts_subsection = "atm_monthly_180x360_aave"
+  # Group 4: ts atm daily
+  # "tropical_subseasonal" (not requested)
+  # Group 5: ts rof monthly
+  # "streamflow" (not requested)
+  # Group 6: tc_analysis
+  # "tc_analysis" (not requested)
+  # Other parameters:
+  diff_title = 'Difference'
+  grid = '180x360_aave'
+  ref_name = 'v3.LR.piControl'
+  ref_start_yr = 0451
+  ref_final_yr = 0500
+  ref_years = "0451-0500",
+  run_type = "model_vs_model"
+  short_name = 'v3.LR.historical'
+  short_ref_name = 'v3.LR.piControl'
+  tag = 'model_vs_model'
+
+  [[ lnd_monthly_mvm_lnd ]]
+  # Depends on the climo task's land_monthly_climo subtask.
+  sets = "lat_lon_land",
+  climo_subsection = "land_monthly_climo"
+  # Other parameters:
+  diff_title = "Difference"
+  grid = 'native'
+  reference_data_path = "/lcrc/group/e3sm2/ac.zhang40/E3SMv3/v3.LR.piControl_451-500/post/lnd/native/clim"
+  ref_name = "20231209.v3.LR.piControl-spinup.chrysalis"
```

Comment:
The source of the second diags error

Reply:
the ref_name should be "v3.LR.piControl", I must have missed this when updating